community/contributing/updating_the_class_reference.rst

```diff
@@ -1,151 +1,176 @@
-
-This command will first ``fetch``, or download the latest version of the Godot repository. Then, it will reapply your local changes on top.
+    origin https://github.com/your_name/godot.git (fetch) origin
+    https://github.com/your_name/godot.git (push) upstream
+    https://github.com/godotengine/godot.git (fetch) upstream
+    https://github.com/godotengine/godot.git (push)
 
-If you made changes you don't want to keep in your local branch, use the following commands instead:
+Each time you want to synchronize your branch with the upstream repository,
+enter:
 
 ::
 
-    git fetch upstream
-    git reset --hard upstream master
-
-**Warning:** The above command will reset your branch to the state of the ``upstream master`` branch. It will discard all local changes. Make sure to only run this *before* you make important changes.
-
-Another option is to delete the branch you're working on, synchronize the master branch with the Godot repository, and create a new branch:
-
-::
-
-    git checkout master
-    git branch -d your-new-branch-name
     git pull --rebase upstream master
-    git checkout -b your-new-branch-name
 
-If you're feeling lost by now, come to our `IRC channels <https://webchat.freenode.net/?channels=#godotengine>`_ and ask for help. Experienced Git users will give you a hand.
+This command will first ``fetch``, that is, download the latest version of the
+Godot repository. Then, it will reapply your local changes on top of it.
 
-Updating the documentation template
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When classes are modified in the source code, the documentation template might become outdated. To make sure that you are editing an up-to-date version, you first need to compile Godot (you can follow the :ref:`doc_introduction_to_the_buildsystem` page), and then run the following command (assuming 64-bit Linux):
+If you made changes you don't want to keep in your local branch, use the
+following commands instead:
 
 ::
 
-    ./bin/godot.linuxbsd.tools.64 --doctool .
+    git fetch upstream git reset --hard upstream master
 
-The XML files in doc/classes should then be up-to-date with current Godot Engine features. You can then check what changed using the ``git diff`` command. If there are changes to other classes than the one you are planning to document, please commit those changes first before starting to edit the template:
+**Warning:** The above command will reset your branch to the state of the
+ ``upstream master`` branch. It will discard all local changes. Make sure to
+ only run this *before* you make important changes.
+
+Another option is to delete the branch you're working on, synchronize the master
+branch with the Godot repository, and create a new branch:
 
 ::
 
-    git add doc/classes/*.xml
-    git commit -m "Sync classes reference template with current code base"
+    git checkout master git branch -d your-new-branch-name git pull --rebase
+    upstream master git checkout -b your-new-branch-name
```

Comment:
Newlines.

Reply:
The `git branch -d` is weird, there's no reason to do that (unless you always use the same branch name for new changes, but that's up to users and we probably don't need to document it. The suggested branch name includes "new".)